### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by SeanNaren**
Formatted decode script, added dels to benchmark and updated train to save best model

### CODE REVIEW

**Comment by SiddGururani** [comment]
About saving the best model, I noticed that you use the WER to determine whether the model is getting better or worse. Shouldn't we be using the loss on the validation set instead? In most of my experiments, I've noticed that the WER might converge to a value but the loss starts increasing after several epochs.

**Comment by SeanNaren** [comment]
@SiddGururani problem is I see that the validation loss diverges after a while, which doesn't usually reflect how well the WER/CER is... I was doing an average over the 2 but I thought WER was representative enough!

File: train.py [hunk 1]
```diff
@@ -372,6 +376,13 @@ def main():
         optimizer.load_state_dict(optim_state)
         print('Learning rate annealed to: {lr:.6f}'.format(lr=optim_state['param_groups'][0]['lr']))
 
+        if best_wer is None or best_wer > wer:
+            print("Found better validated model, saving to %s" % args.final_model_path)
```
Comment (mit456) [inline train.py]: Shouldn't it be `args.model_path`?

Comment (SeanNaren) [inline train.py]: Thanks for that, fixed [here](https://github.com/SeanNaren/deepspeech.pytorch/commit/3ffa9a5c2d93000d473208cc5506decc8949f8b3)
